guess-fav-num.js

```diff
@@ -0,0 +1,34 @@
+/* exported checkGuessNumber, resetGuessNumber, guessNumberResponse*/
+/* use strict */
+var guessNumberForm = document.getElementById('guess-number-form');
+var guessNumberResponse = document.getElementById('guess-number-response');
+
+var guessCount = 0;
+
+function checkGuessNumber() {
+    var elements = guessNumberForm.elements;
+    var guess = elements.number.value;
+    var submit = elements.guess;
+
+    if(guess !== 5) {
```

Comment:
```suggestion
    if(guess === '5') {
```
I believe you meant to compare equal to rather than not equal to (=== vs !==). Also your conditional statement should be if the guess is strictly equal to the _string_ '5' (not number 5) as that is the input value.